### PR TITLE
some update for postgres driver validation

### PR DESCRIPTION
**Pull request by narutimateum**
-added pgsql in driver validation
-update gitignore

### CODE REVIEW

**Comment by narutimateum** [comment]
please check and approve.. i need this urgently

File: .gitignore [hunk 1]
```diff
@@ -3,3 +3,4 @@
 composer.lock
 cache
 *.sqlite
+.idea
```
Comment (theofidry) [inline .gitignore]: this is unnecessary, it should be in your [global gitignore](https://stackoverflow.com/questions/7335420/global-git-ignore) file (check https://www.gitignore.io/ for OS/IDE related things)

**Comment by theofidry** [comment]
If you need this urgently I recommend you to just make use of your fork (which can be used immediately) instead of waiting a review + merge + release.

**Comment by theofidry** [comment]
@wouterj the failure of Travis is unrelated (it's due to PHPUnit 6.x)

**Comment by wouterj** [comment]
Thanks for discovering this bug! I have absolutely no idea why I used `postgres` in this bundle... (it was probably used in older Laravel versions).

I discovered `sql server` is also wrong. So I took your commit, added the `sqlsrv` name and triggered a deprecation warning for the old names. That resulted in a new PR: https://github.com/wouterj/WouterJEloquentBundle/pull/45

**Pull request closed without merging**